src/create-react.ts

```diff
@@ -47,64 +47,64 @@ function getAllComponents(metadata): Component[] {
 export const createReactWrapper = (infos: ReactWrapperMetadata, prefix: string) => {
   const componentPrefix = pascalCase(prefix);
   return `
-${ infos.customEvents ? "import type {EventName} from '@lit-labs/react';" : "" }
+${infos.customEvents ? 'import type {EventName} from \'@lit-labs/react\';' : ''}
 import * as React from 'react';
 import { createComponent } from '@lit-labs/react';
-${ infos.imports.join("\n") }
+${infos.imports.join('\n')}
 
-${ infos.components.map(component => {
+${infos.components.map(component => {
     return `
-export const ${ componentPrefix }${ component.name }Component = createComponent(
+export const ${componentPrefix}${component.name}Component = createComponent(
   React,
-  '${ component.tagName }',
-  ${ component.name }Component,
+  '${component.tagName}',
+  ${component.name}Component,
   {
-    ${ component.events }
+    ${component.events}
   }
 );
 `;
-  }).join("\n") }
+  }).join('\n')}
 `;
 };
 
 export const createReactWrapperMetadata = (metadata: Record<string, unknown>, prefix: string, getComponentPath: (name: string) => string): ReactWrapperMetadata => {
   const components = getAllComponents(metadata);
   const createEvent = (event) => `${event.reactName}: '${event.name}' as EventName<${event.type.text}>`;
 
-
   return components.reduce((obj: ReactWrapperMetadata, component): ReactWrapperMetadata => {
-    const tagWithoutPrefix = component.tagName.replace(prefix, "");
+    const tagWithoutPrefix = component.tagName.replace(prefix, '');
     const sourceName = path.parse(component.modulePath).name;
     const importPath = getComponentPath(sourceName);
-    const events = (component.events || []).map(createEvent).join(",\n");
+    const events = (component.events || []).map(createEvent).join(',\n');
     const imports = (component.events || [])
       .filter(event => !event.type.text.startsWith('CustomEvent'))
+      .filter(event => event.name.startsWith(prefix))
```

Comment:
Sorry for all the changes in single/double quotes, that happend on reformat. This is the real change. Only add imports to the events that start with our prefix.